src/Microsoft.DotNet.XHarness.Apple/AppTester.cs

```diff
@@ -450,28 +458,29 @@ public class AppTester : AppRunnerBase
             return await testReporter.ParseResult();
         }
 
-        private Dictionary<string, object> GetEnvVariables(
+        private Dictionary<string, string> GetEnvVariables(
             XmlResultJargon xmlResultJargon,
             string[]? skippedMethods,
             string[]? skippedTestClasses,
             ListenerTransport listenerTransport,
             int listenerPort,
-            string listenerTmpFile)
+            string listenerTmpFile,
+            IEnumerable<(string, string)> extraEnvVariables)
         {
-            var variables = new Dictionary<string, object>
+            var variables = new Dictionary<string, string>
             {
-                { EnviromentVariables.AutoExit, true },
-                { EnviromentVariables.HostPort, listenerPort },
+                { EnviromentVariables.AutoExit, "true" },
```

Comment:
Why did we move to strings? I fear parsing annoying code.

Reply:
@mandel-macaque they become strings eventually, only that I found that booleans become capitalized (`True`/`False`) so it seems to have more control over this from the beginning